src/main/scala/io/github/spark_redshift_community/spark/redshift/Conversions.scala

```diff
@@ -37,7 +37,8 @@ private[redshift] object Conversions {
     * "A formatter created from a pattern can be used as many times as necessary,
     * it is immutable and is thread-safe."
     */
-  private val formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss[.SSS][.SS][.S][X]")
+  private val formatter = DateTimeFormatter.ofPattern(
+    "yyyy-MM-dd HH:mm:ss[.SSSSSS][.SSS][.SS][.S][X]")
```

Comment:
I think `[.SSSS]` and `[.SSSSS]` are needed too, since Redshift timestamps can have any number of decimal places:
```
> select '2019-01-01 00:00:00.0001' :: timestamp                                                                                                                                                                              
+--------------------------+
| timestamp                |
|--------------------------|
| 2019-01-01 00:00:00.0001 |
+--------------------------+
```
I tried this in an `unload` and the format was the same in the file on S3.

Reply:
Right, I'll add them, thanks for pointing this out